etc/cvmfs/default.conf

```diff
@@ -1,9 +1,13 @@
-if [ "$CVMFS_HTTP_PROXY" = "DIRECT" ] || [ "$CVMFS_HTTP_PROXY" = "auto;DIRECT" ]; then
-    CVMFS_SERVER_URL="http://s1bnl-cvmfs.openhtc.io/cvmfs/@fqrn@;http://s1fnal-cvmfs.openhtc.io/cvmfs/@fqrn@;http://s1osggoc-cvmfs.openhtc.io/cvmfs/@fqrn@"
-    CVMFS_FALLBACK_PROXY=""
-else
-    CVMFS_SERVER_URL="http://cvmfs-s1bnl.opensciencegrid.org:8000/cvmfs/@fqrn@;http://cvmfs-s1fnal.opensciencegrid.org:8000/cvmfs/@fqrn@;http://cvmfs-s1goc.opensciencegrid.org:8000/cvmfs/@fqrn@"
+if [ -z "$CVMFS_PAC_URLS" ] || [ "$CVMFS_PAC_URLS" = "http://wpad/wpad.dat" ]; then
```

Comment:
Should we add this guard to the cvmfs-config.cern.ch repository, too?

Reply:
Yes I think so.  Also I think there should be a check there to not set fallback proxies if the user set CVMFS_USE_CDN=yes.  I plan to make a PR after this one for the egi branch, and then another for the default branch to make as much in common between the branches as possible while preserving their intentional differences.